app/templates/components/request/configs-buttons.hbs

```diff
@@ -0,0 +1,37 @@
+<div class="request-configs-buttons" data-test-request-configs-button='true'>
+  <a
+    data-test-request-configs-button-cancel="true"
+    class="request-configs-button request-configs-button-cancel"
+    {{on 'click' (action this.onCancel)}}
+  >
+    Cancel
+  </a>
+  <a
+    data-test-request-configs-button-customize="true"
+    class="request-configs-button request-configs-button-customize"
+    {{on 'click' (action this.onCustomize)}}
+  >
+    Customize
+  </a>
+  <a
+    data-test-request-configs-button-preview="true"
+    class="request-configs-button request-configs-button-preview"
+    {{on 'click' (action this.onPreview)}}
+  >
+    Preview
+  </a>
```

Comment:
I'd like to suggest transforming these buttons to be `<button>`s, not links. Links suppose to navigate somewhere. We have multiple classes [here](https://github.com/travis-ci/travis-web/blob/180f93652338b17fbcea4352eef48faf2c2eed6f/app/styles/app/modules/buttons.scss) that can help to reset default button styles. And if you need a cursor to be `pointer`, there's a class `.pointer` that will do the trick.

Reply:
sure thing! done: https://github.com/travis-ci/travis-web/commit/0c3332ab3a57adcec56b6724d389b184aaf89776

Reply:
Awesome, thanks! 😄